srcalc/src/test/java/gov/va/med/srcalc/web/WebConfigIT.java

```diff
@@ -48,9 +40,5 @@ public static void setupJndi() throws NamingException
     public void testWebConfig()
```

Comment:
Is there a way to make this test more useful, or is just loading the application context good enough?